src/Twig/Runtime/RecordRuntime.php

```diff
@@ -134,6 +134,35 @@ public function excerpt($content, $length = 200, $focus = null)
         return $excerpt;
     }
 
+    /**
+     * Find the first image in the given content record.
+     *
+     * This method is deprecated. In Bolt 3.5 it'll be replaced / removed to
+     * fall in line with the work done for #6985
+     *
+     * @param \Bolt\Storage\Entity\Content $content
+     *
+     * @return array image
```

Comment:
`@return array|null`

Reply:
I was just following what other methods in that class did, like https://github.com/bolt/bolt/blob/3.4/src/Twig/Runtime/RecordRuntime.php#L69 and https://github.com/bolt/bolt/blob/3.4/src/Twig/Runtime/RecordRuntime.php#L152. But, i'll happily change it.

Reply:
Yeah, there is a huge lack of consistency … it wasn't one I was terribly worried about, just threw it in with the other comments.